projects/lib/src/knockouttournament.cpp

```diff
@@ -198,8 +198,8 @@ void KnockoutTournament::setTC(TournamentPlayer white, TournamentPlayer black, C
 		return;
 	}
 
-	int firstScore  = pair->firstScore() + white.builder()->resumescore();
-	int secondScore = pair->secondScore() + black.builder()->resumescore();
+//	int firstScore  = pair->firstScore() + white.builder()->resumescore();
+//	int secondScore = pair->secondScore() + black.builder()->resumescore();
```

Comment:
It's better to remove unnecessary code than to comment it out. Anyway, this should probably be in a separate clean-up patch.